src/schedule-stream/utils.js

```diff
@@ -1,7 +1,7 @@
 const {getChannel} = require('../utils')
 
 function getStreamerChannel(guild) {
-  return getChannel(guild, {name: '📅-stream-schedule'})
+  return getChannel(guild, {name: '⏱-upcoming-streams'})
```

Comment:
In case we change the emoji, let's leave that out:

```suggestion
  return getChannel(guild, {name: 'upcoming-streams'})
```